app/shared/layouts/navigation/NavigationProvider.js

```diff
@@ -1,68 +1,95 @@
 'use client';
 import { createContext, useCallback, useEffect, useState } from 'react';
+import { FaDiscord } from 'react-icons/fa';
+
 import {
-  FaClipboardList,
-  FaDiscord,
-  FaGraduationCap,
-  FaListUl,
-  FaPeopleCarry,
-  FaRegCalendarCheck,
-  FaRegNewspaper,
-  FaSearch,
-  FaTools,
-  FaUserTie,
-} from 'react-icons/fa';
+  ConstructionRounded,
+  EmojiEventsRounded,
+  GroupRounded,
+  FolderSharedRounded,
+  LibraryBooksRounded,
+  SlideshowRounded,
+  SearchRounded,
+  VolunteerActivismRounded,
+  EventAvailableRounded,
+  MailRounded,
+  NewspaperRounded,
+  ListRounded,
+} from '@mui/icons-material';
```

Comment:
We're trying to stick to utilizing `react-icons` as much as we can so can you see if these can be pulled from `react-icons/md` https://react-icons.github.io/react-icons/icons/md/ ? 🙏

Reply:
Well.. apparently we're using MUI icons now.. 🤷